af/fastpath/fastpath/db.py

```diff
@@ -227,6 +81,45 @@ def _click_create_table_fastpath():
     log.debug(list(rows))
 
 
+def click_create_table_obs_openvpn():
+    sql = """
+    CREATE TABLE IF NOT EXISTS obs_openvpn
+    (
+        anomaly Bool,
+        bootstrap_time Float32,
+        confirmed Bool,
+        error String,
+        failure Bool,
```

Comment:
According to the spec this field is a Nullable string

Reply:
https://github.com/ooni/spec/pull/255/files#diff-bf7281a292d49fde7131b3803a9d7f1d7301fa875dec3a907d6594eac299f273R238

Reply:
On the DB side failure is a boolean and the value is casted into bool by https://github.com/ooni/pipeline/pull/399/files#diff-ba56dd346dd6041329c79d115e02ad095b974f2bbe6cac021d916fead8e9432fR298

Reply:
I think that's quite confusing and much less useful than storing the actual failure string. You can always get the bool from checking if the string is not NULL.